native/package.json

```diff
@@ -36,22 +36,17 @@
     "ts:check": "tsc --build"
   },
   "dependencies": {
-    "@react-native-community/art": "^1.1.2",
```

Comment:
This could be a peer dependency. While it is not required to specify them, it is actually good. I'm not sure whether any of the dependencies below are peer dependencies.

Reply:
I checked for all dependencies, none of them are peer dependencies or dependencies of other modules.

Reply:
events is actually needed for htmlparser2 though.